doc/project/project_roles.rst

```diff
@@ -314,15 +314,37 @@ Release Activity
 Merge Criteria
 ++++++++++++++
 
-* All continuous integration checks have passed
+* Minimal of 2 approvals:
+
+  * A collaborator from the same subsystem.
+  * Alternately another maintainer of another subsystem
+  * Approval by the assignee
```

Comment:
This is a bit unclear, should it be at least two approvals out of these three categories, or two approvals coming from one category is ok ?

Reply:
at least two approvals from all categories, I can make this clearer